### PR TITLE
Permissionlessness vs Permissiveness

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
⚠️⚠️⚠️ **Please do not merge until 2022-12-08** (I can actually merge it when it is reviewed and approved.)

In this opinion article I explain why for ETC to be permissionless the ETC community must be permissioned. Including all its online assets and the ECIP process.

ZH version coming soon.

### CODE REVIEW

**Comment by DonaldMcIntyre** [comment]
@IstoraMandiri @w1g0 @TheCrowbill, I received criticism for the title of this post "Permissionlessness Must not Permit Permissionness" but I think it is a very good and challenging phrase, and the article explains everything in the body. What do you guys think? Should I change it to something easier, like, "The Permissionless Paradox" or the "The Permissionless Fallacy"?

**Comment by TheCrowbill** [comment]
Permissionness isn't an English word. That said, as a coined word used in a provocative title and fully explained in the text, I have no problem with such a usage though others may.

As an alternative to using a coined word, I would suggest rewording using "permissiveness" instead.

**Comment by DonaldMcIntyre** [comment]
> As an alternative to using a coined word, I would suggest rewording using "permissiveness" instead.

That is a good idea. I will see if the change to "Permissiveness" passes the test! (Yes, as a non-english speaker I coined "permissionness" as I didn't remember that "permissiveness" exited! 😄).
